core/src/main/resources/defaults/blueprints/7.2.11/cdp-data-engineering-ha.bp

```diff
@@ -182,6 +182,10 @@
               {
                 "name": "hiveserver2_idle_session_timeout",
                 "value": 14400000
+              },
+              {
+                "name": "hive.privilege.synchronizer",
```

Comment:
Setting it this way is not correct. Its not applied to HS2. If you open hive-site.xml from UI, you wont see it getting set anywhere. You wont see it getting set via beeline either

0: jdbc:hive2://perf-test2-master0.perf-aws.x> set hive.privilege.synchronizer;
+-----------------------------------+
|                set                |
+-----------------------------------+
| hive.privilege.synchronizer=true  |
+-----------------------------------+

Right way to set this is as you did it first time. Add this property to hive_service_config_safety_valve